caput/algorithms/invert_no_zero/__init__.py

```diff
@@ -12,24 +17,26 @@
 ]
 
 
-def invert_no_zero(x, out=None):
+def invert_no_zero(
+    x: npt.ArrayLike[np.number], out: npt.ArrayLike[np.number] | None = None
+) -> npt.ArrayLike[np.number]:
     """Return the reciprocal, but ignoring zeros.
 
     Where `x != 0` return 1/x, or just return 0. Importantly this routine does
```

Comment:
```suggestion
    Where ``x != 0`` return ``1/x``; where ``x == 0``, return 0. Importantly this routine does
```

I think this needs to be more explicit.